controller/work_item_link_type.go

```diff
@@ -18,8 +18,9 @@ import (
 // WorkItemLinkTypeController implements the work-item-link-type resource.
 type WorkItemLinkTypeController struct {
 	*goa.Controller
-	db     application.DB
-	config WorkItemLinkTypeControllerConfiguration
+	db           application.DB
+	config       WorkItemLinkTypeControllerConfiguration
+	configCombis WorkItemLinkTypeCombinationControllerConfiguration
```

Comment:
sure but you could have a configuration type with multiple methods, too. I'm not sure it's clean to have a controller with multiple configurations here, honestly.